packages/build/src/plugins_core/functions/index.ts

```diff
@@ -10,10 +10,15 @@ import { getZipError } from './error.js'
 import { getUserAndInternalFunctions, validateFunctionsSrc } from './utils.js'
 import { getZisiParameters } from './zisi.js'
 
-// Returns `true` if at least one of the functions has been configured to use
-// esbuild.
-const isUsingEsbuild = (functionsConfig = {}) =>
-  Object.values(functionsConfig).some((configObject) => configObject.node_bundler === 'esbuild')
+// Get a list of all unique bundlers in this run
+const getBundlers = (results: Awaited<ReturnType<typeof zipFunctions>> = []) =>
```

Comment:
nit: I find all this nesting a bit hard to read. I would at least move the `Array.from()` call to the caller in L53 and make this return a `Set`, but it's totally up to personal preference so feel free to ignore.

Reply:
Done